templates/article_stub.html

```diff
@@ -13,7 +13,6 @@ <h2 class="title">
                 <div class="clear"></div>
 
                 <div class="info">
-                    <a href="{{ SITEURL }}/{{ article.url }}">posted at {{ article.date.strftime("%H:%M") }}</a>
```

Comment:
Sorry, it's unrelated to the pull request but I already commited the change before working on the disqus integration.